easybuild/easyconfigs/i/IGV/IGV-2.5.0-Java-11.eb

```diff
@@ -0,0 +1,35 @@
+# EasyBuild easyconfig
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# Modified by Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'Tarball'
+
+name = 'IGV'
+version = '2.5.0'
+versionsuffix = '-Java-11'
+
+homepage = 'http://www.broadinstitute.org/software/igv/'
+description = """This package contains command line utilities for
+ preprocessing, computing feature count density (coverage),  sorting, and
+ indexing data files."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://data.broadinstitute.org/igv/projects/downloads/' +
+               '%(version_major)s.%(version_minor)s']
+sources = ['%(name)s_%(version)s.zip']
+checksums = ['44efdd0500dd6ace8e1c70dad0d758d9ab90492b391d2af4a416f03808fee471']
+
+dependencies = [('Java', '11.0.2', '', True)]
```

Comment:
@fizwit Please depend on the `Java/11` wrapper, rather than on a specific version:

```python
dependencies = [('Java', '11', '', True)]
```

Reply:
Yow, my bad, fixed with updated pr